addons/hr/models/res_partner.py

```diff
@@ -39,3 +39,15 @@ def action_open_employees(self):
             'view_mode': 'kanban,tree,form',
             'domain': [('id', 'in', self.employee_ids.ids)],
         }
+
+class ResPartnerBank(models.Model):
+    _inherit = ['res.partner.bank']
+
+    @api.depends_context('uid')
```

Comment:
This is causing cache misses and throws a big error when you create an create a vendor bill from an employee expense and a user who doesn't have the `hr.group_hr_user` opens the vendor bill
![image](https://github.com/user-attachments/assets/618fd1c9-3978-45c7-bee9-e7929b5672a9)

I think it's because the initial cache miss causes a compute with `sudo()` but because of the `depends_context`, the cache key misses because it's a `(uid, su)` tuple.

Would recommend removing this line

Reply:
oh thanks for the notice, let me investigate this! (or if you wish to make a pr)

Reply:
@hwangh95 could you have a look at #187772 and let me know if it fixes your issue?